README.md

```diff
@@ -172,17 +187,24 @@ Replace `OUTPUT_PATH` with the directory where you want Diana to store its Helm
 diana configure-mq-backend ~/neon_diana
 ```
 
-Follow the prompts to provide any necessary configuration parameters. 
+Follow the prompts to provide / add any necessary configuration parameters. 
+In order to enable all Neon skills, you will need to enable / choose yes for all these services. 
+Tips:
+Visit https://github.com/settings/tokens to generate the necessary token for the GitHub private services. 
```

Comment:
Prefix each item with `- ` to render as a list, otherwise this will just format as a continuous block of text